lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -84,9 +80,19 @@ function collectAllStringsInDir(dir, strings = {}) {
           let lastPropertyEndIndex = 0;
           for (const property of stmt.declarations[0].init.properties) {
             const key = property.key.name;
-            const message = exportVars.UIStrings[key];
-            const description = computeDescription(ast, property, lastPropertyEndIndex);
-            strings[`${relativePath} | ${key}`] = {message, description};
+            const val = exportVars.UIStrings[key];
+            const res = collUtil.computeDescription(ast, property, val, lastPropertyEndIndex);
+
+            const description = res.description;
+            const examples = res.examples;
+
+            const converted = collUtil.convertMessageToPlaceholders(val, examples);
+
+            if (Object.entries(converted.placeholders).length === 0 && converted.placeholders.constructor === Object) {
+              strings[`${relativePath} | ${key}`] = {message: converted.message, description};
```

Comment:
nit: maybe we could set the object with the if and/or compute the special key in only one place?